src/components/AdminPortal/Advertisements/Advertisements.module.css

```
@@ -0,0 +1,56 @@
+  font-weight: var(--font-weight-medium);
+  line-height: var(--line-height-tight);
+  padding-bottom: var(--space-7);
+}
+
+.justifyspAdvertisements {
+  display: grid;
+  width: 100%;
+  margin-top: var(--space-7-5);
+  grid-template-rows: auto;
+  grid-template-columns: repeat(6, 1fr);
+  grid-gap: var(--space-4) var(--space-3-5);
```

Stylelint (17.6.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)